### PR TITLE
feat/partner-with-us

### PR DESCRIPTION
**Pull request by misspelz**
image

### CODE REVIEW

File: src/assets/partnerwithus.png
Comment (edustipend-root) [inline src/assets/partnerwithus.png:1]: @misspelz this image is quick large, almost 1MB - that's a lot to be adding to the size of this app. 

Can we get in touch with the designer Ubon and see if we can have a compressed image that isn't this heavy?

File: src/sections/Impacts/internals/PartnerWithUs.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          <p data-testid={TestId.PARTNER_SUB_HEADER_TEXT} className={styles.partnersubheader}>
+            {Texts.PARTNER_SUB_HEADER}
+          </p>
+
+          <div className={styles.cta}>
+            <NavHashLink
+              to={{
+                pathname: '/support-a-learner/donate'
+              }}>
+              <Button label={'Support a Learner'} size={'large'} type={'secondary'} />
```
Comment (edustipend-root) [inline src/sections/Impacts/internals/PartnerWithUs.jsx:27]: Please put the `'Support a learner' text into a `constant.js` file.

**Review by edustipend-root — CHANGES_REQUESTED** [review]
Changes look good @misspelz  but some minor comments - let's tweak this and clean it up

File: src/assets/partnerwithus.png
Comment (misspelz) [reply src/assets/partnerwithus.png:1]: I just compressed to less than 400mb. How's that?

File: src/sections/Impacts/internals/PartnerWithUs.jsx (hunk 1 above)
Comment (misspelz) [reply src/sections/Impacts/internals/PartnerWithUs.jsx:27]: Done

**Review by edustipend-root — APPROVED** [review]

**Pull request merged**